schematic_api/api/routes.py

```diff
@@ -1,39 +1,37 @@
 from email import generator
```

Comment:
This might be improved when using Black later, but in general we should be keeping imports in alphabetical order. They don't all need to be changed now, but when adding new imports try to keep this in mind.

Reply:
hmm my understanding is that import should be organized based on the order: general standard libraries comes first, then 3rd party, then local packages. This is mentioned here: https://peps.python.org/pep-0008/#imports:~:text=Imports%20should%20be%20grouped%20in%20the%20following%20order%3A I am sure there are third party tools that could organize imports if black is not sufficient. We could discuss this in the code review

Reply:
`from emails import generator` seems a bit strange. I removed it because it doesn't seem like this is being used